frontend/javascripts/dashboard/folders/details_sidebar.tsx

```diff
@@ -173,12 +173,22 @@ function DatasetDetails({ selectedDataset }: { selectedDataset: APIDatasetCompac
           )}
         </div>
       </Spin>
+
       {selectedDataset.isActive ? (
         <div style={{ marginBottom: 4 }}>
           <div className="sidebar-label">Tags</div>
           <DatasetTags dataset={selectedDataset} updateDataset={context.updateCachedDataset} />
         </div>
       ) : null}
+
+      {fullDataset?.usedStorageBytes && fullDataset.usedStorageBytes > 10000 ? (
+        <div style={{ marginBottom: 4 }}>
+          <div className="sidebar-label">Used Storage</div>
+          <Tooltip title={`${fullDataset.usedStorageBytes} Bytes`} placement="left">
```

Comment:
I think, showing the bytes in the tooltip doesn't really add a value. I'd rather say something like "Note that linked layer's aren't measured" or something like that.